### PR TITLE
Fix calendar icon to fit properly after field #1428

### PR DESCRIPTION
**Pull request by wiadev**
- open calendar by default after double-click

#1428

### CODE REVIEW

File: src/components/widget/DatePicker.js [hunk 1]
```diff
@@ -87,15 +87,17 @@ class DatePicker extends Component {
...
-      <div tabIndex="-1" onKeyDown={this.handleKeydown}>
+      <div tabIndex="-1" onKeyDown={this.handleKeydown} className="datepicker">
         <Datetime
           closeOnTab={true}
           renderDay={this.renderDay}
           renderInput={this.renderInput}
           onBlur={this.handleBlur}
           onFocus={this.handleFocus}
           {...this.props}
+          open={true}
```
Comment (metas-mk) [inline src/components/widget/DatePicker.js]: @wiadev seems as if the datepickers are now always initially opened and don't close anymore.

see the following example
image

Comment (metas-mk) [inline src/components/widget/DatePicker.js]: Also in edit mode. After doubleclick on the datefield to switch into edit more, the datepicker does not dissappear after selection anymore.
image

**Comment by wiadev** [comment]
@metas-mk updated

**Comment by wiadev** [comment]
@metas-mk can you check out this, please? I might need this integrated into #1387.

**Comment by wiadev** [comment]
please, check #1507, first before this PR.

**Comment by metas-mk** [comment]
Quick local check OK. Integrating.

**Pull request merged**